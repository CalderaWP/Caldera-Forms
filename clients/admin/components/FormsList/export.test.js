import FormListComponents from './index';
import {snapshotObjectKeysAndTypes} from "../../testUtil/snapshotObjectKeysAndTypes";

describe( 'Form List components export', () => {
	it( 'matches snapshot', () => {
		snapshotObjectKeysAndTypes(FormListComponents);
	});
});