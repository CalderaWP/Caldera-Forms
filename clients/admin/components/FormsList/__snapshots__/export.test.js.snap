// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form List components export matches snapshot 1`] = `"{\\"Form\\":\\"function\\",\\"FormList\\":\\"function\\",\\"ShortcodeViewer\\":\\"function\\"}"`;
